subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/page_object/Alert.kt

```diff
@@ -1,12 +1,21 @@
 package com.avito.android.test.page_object
 
+import androidx.appcompat.widget.AlertDialogLayout
 import androidx.appcompat.widget.DialogTitle
-import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.isAssignableFrom
 import androidx.test.espresso.matcher.ViewMatchers.withId
+import com.avito.android.test.InteractionContext
+import com.avito.android.test.context.DialogInteractionContext
 
 class Alert : PageObject() {
+
+    override val interactionContext: InteractionContext
```

Comment:
Should be marked as a `final`?

Reply:
Added, just in case.
Now it's not effective, because the class is not open.
This is ok for me to show the intention.